handlers/toggle_interface.yml

```diff
@@ -24,6 +24,6 @@
 
 - name: reapply any interface with ipv4 addr to get all configurations activated
   shell: "nmcli connection up 'System {{ item }}'"
-  when: ansible_facts[item].ipv4.address is defined and not 'lo' == item
+  when: ansible_facts[item].ipv4.address is defined and not 'lo' == item and 'bond' not in item
```

Comment:
as discussed f2f, we have also some nodes for which the bridge interfaces has an ipv4 address, so bridge interfaces should be probably excluded too

Reply:
just tried running the role against a node for which the bridge interface had an ipv4 address. No errors were actually thrown, so no need for changes